### PR TITLE
Update required packages

### PR DESCRIPTION
**Pull request by XiaojuanHu**
review pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ FFAFFURR is a Python tool, which enables the parametrization of OPLS-AA and CTPO
 - python 3 
 - OpenMM 7.4.2
 - [pyswarm](https://github.com/XiaojuanHu/pyswarm)
+- Numpy
+- Pandas
```
Comment (shudipto-amin) [inline README.md]:
```suggestion
- [Numpy](https://numpy.org/)
- [Pandas](https://pandas.pydata.org/)
```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ FFAFFURR is a Python tool, which enables the parametrization of OPLS-AA and CTPO
...
 
 ## Inputs
```
Comment (shudipto-amin) [inline README.md:13]: Maybe "input files" instead?

**Review by shudipto-amin — COMMENTED** [review]
Looks good to me, see comments.

File: README.md (hunk 2 above)
Comment (XiaojuanHu) [reply README.md:13]: Okay

**Pull request merged**